### PR TITLE
Docs + Release Announcment for 1.20-1

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*

# HOLD UNTIL RELEASE

### TODO

- [ ] Update changelog with component versions
- [ ] Update index.md with crd
- [ ] Update release announcement with specifics
- [ ] Update pull-all.sh with versions for 1-20

All of these todo items will be resolved once we figure out the component versions

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/DEFAULT_RELEASE_BRANCH [hunk 1]
```diff
@@ -1 +1 @@
-1-19
+1-20
```
Comment (kschumy) [inline release/DEFAULT_RELEASE_BRANCH:1]: Should we make this change in a different PR?

Comment (TerryHowe) [reply release/DEFAULT_RELEASE_BRANCH:1]: I'm fine with that

File: development/pull-all.sh [hunk 2]
```diff
@@ -1,21 +1,42 @@
...
-docker pull public.ecr.aws/eks-distro/kubernetes/kube-apiserver:v1.18.9-eks-1-18-1
+set -e
+set -x
+
+RELEASE_BRANCH=1-20
+RELEASE=1
+RELEASE_TAG="eks-${RELEASE_BRANCH}-${RELEASE}"
+KUBERNETES_VERSION=v1.20.4
+
+docker pull public.ecr.aws/eks-distro/kubernetes/pause:${KUBERNETES_VERSION}-${RELEASE_TAG}  # TODO: want the hard coded version?
```
Comment (kschumy) [inline development/pull-all.sh:24]: Do we want to hard code this value in? Previously it wasn't hard coded here

Comment (TerryHowe) [reply development/pull-all.sh:24]: No, we should pull these out of the associated $RELEASE_BRANCH/GIT_TAG file at some point. Putting this change in the 1.20 docs kind of clutters this PR up, but I can live with what you have here and it can be fixed in a follow up.

**Comment by kschumy** [comment]
Closing because I ended up having to change a lot, there were merge conflicts, and it was bad. Reopening in different PR on different branch
